demo/full/styles/style.css

```diff
@@ -14,7 +14,7 @@ body {
 }
 
 .title-wrapper h1 {
-  font-family: "Open Sans", sans serif;
+  font-family: "Open Sans", sans-serif;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected missing generic font family ([css:S4649](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4649))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=264&issues=AWTMwmjYi6fIhhPMqNRT&open=AWTMwmjYi6fIhhPMqNRT)

[ISSUE_KEY]: AWTMwmjYi6fIhhPMqNRT